apps/omg_api/test/block_queue/core_test.exs

```diff
@@ -48,14 +48,15 @@ defmodule OMG.API.BlockQueue.CoreTest do
   end
 
   deffixture empty_with_gas_params(empty) do
-    state = %{empty | formed_child_block_num: 5, gas_price_to_use: 100}
+    state = %{empty | formed_child_block_num: 5 * @child_block_interval, gas_price_to_use: 100}
 
     {:dont_form_block, state} =
       state
-      |> set_ethereum_status(1, 3, false)
+      |> set_ethereum_status(1, 3 * @child_block_interval, false)
 
     # assertions - to be explicit how state looks like
-    assert {1, 3} = state.gas_price_adj_params.last_block_mined
+    child_block_mined = 3 * @child_block_interval
+    assert {1, ^child_block_mined} = state.gas_price_adj_params.last_block_mined
```

Comment:
oops, testing by reaching into the guts. Is there a behavior we can test here (old problem)?

**EDIT** I see more occurrences of this, so let's split out refactoring the tests to a different PR/ticket